### PR TITLE
Update test files.

### PR DESCRIPTION
**Pull request by m-chau**
fix some minor bugs in the test files.

### CODE REVIEW

**Comment by shalinijoshi19** [comment]
:shipit:

File: org.opent2t.sample.thermostat.superpopular/test/thermostatTests.js [hunk 1]
```diff
@@ -100,7 +100,7 @@ function runThermostatTests(settings) {
     test.serial('SetTargetTemperatureHigh', t => {
         setData(t);
         return OpenT2T.invokeMethodAsync(translator, SchemaName, 'getDevicesTargetTemperatureHigh', [deviceId]).then((initialTemperature) => {
-            return OpenT2T.invokeMethodAsync(translator, SchemaName, 'postDevicesTargetTemperatureHigh', [deviceId, {'temperature': 22}]).then(() => {
+            return OpenT2T.invokeMethodAsync(translator, SchemaName, 'postDevicesTargetTemperatureHigh', [deviceId, { 'temperature': 22, 'units': 'c' }]).then(() => {
```
Comment (shalinijoshi19) [inline org.opent2t.sample.thermostat.superpopular/test/thermostatTests.js]: >{ ' [](start = 116, length = 3)

Interesting so teh spacing matters?

**Comment by danielrischar** [comment]
:shipit:

**Comment by arjun-msft** [comment]
:shipit:

File: org.opent2t.sample.thermostat.superpopular/test/thermostatTests.js (hunk 1 above)
Comment (m-chau) [inline org.opent2t.sample.thermostat.superpopular/test/thermostatTests.js]: Not really, I just needed the "units" object for Nest. I think the spaces were added by VS automatically. Should I remove them?

---
In reply to: [98082435](https://github.com/openT2T/translators/pull/150#discussion_r98082435) [](ancestors = 98082435)

**Pull request merged**